spec/templater_spec.rb

```diff
@@ -164,11 +164,13 @@ def expect_environment_dependent_parameters_present(magic_namespace)
       expect(genders_config.non_existent).to eq([])
     end
 
-    before do
-      # Stub this so mock hunter config used.
-      stub_const('Metalware::Constants::HUNTER_PATH', TEST_HUNTER_PATH)
+    let :filesystem {
+      FileSystem.setup do |fs|
+        fs.with_repo_fixtures 'repo'
+      end
+    }
 
-      SpecUtils.use_unit_test_config(self)
+    before do
       SpecUtils.use_mock_determine_hostip_script(self)
       SpecUtils.use_mock_genders(self)
```

Comment:
This is probably a future change, but do we need to mock out the `genders` and the `hostip_scripts` anymore if we are using FakeFS? It would be cleaner if we could integrate them with `FileSystem` if appropriate.